container-registry-configuration/README.md

```diff
@@ -1,6 +1,12 @@
 ---
 creation_date: 2023-03-01
+issues:
+- https://github.com/giantswarm/giantswarm/issues/24237
+last_review_date: 2024-02-26
+owners:
+- https://github.com/orgs/giantswarm/teams/team-turtles
```

Comment:
I am not sure about this. I created this RFC and Rocket and Phoenix implemented this in cluster-<provider> charts. Now, @giantswarm/team-turtles is responsible for this kind of provider-independent thing. 

I wonder if I should write my name. @AndiDog

Reply:
Fine to have only Team Turtles here, since they'll need to make the containerd config consistent across providers.